spec/models/factory_girl_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe 'FactoryGirl' do
+  local_models = Dir.glob('app/models/*.rb').map { |file| File.basename(file, '.rb').to_sym }
+  factories = FactoryGirl.factories.collect(&:name)
+
+  ApplicationRecord.descendants.each do |klass|
+    model = klass.name.underscore.to_sym
+    next unless local_models.include?(model) # Skip non-locally defined models
+
+    context klass.name do
```

Comment:
Do we really need a context for each local model?  I feel like this can be a single expectation

Reply:
I already answered this here: https://github.com/ManageIQ/manageiq/pull/16404#discussion_r149298481

Reply:
I suggested this very thing as well in [my review](https://github.com/ManageIQ/manageiq/pull/16404#pullrequestreview-74578400). I agree with the points @bdunne made; and I don't think being afraid of someone just changing the test to pass is valid, to be honest ;) You could do the same with this current version, too.

Reply:
@chrisarcand @bdunne okay